taskfile/node.go

```diff
@@ -22,7 +22,7 @@ type Node interface {
 	Remote() bool
 	ResolveEntrypoint(entrypoint string) (string, error)
 	ResolveDir(dir string) (string, error)
-	FilenameAndLastDir() (string, string)
+	FilenameAndLastDir() (lastDir string, file string) // TODO the return order is implemented opposite to the naming
```

Comment:
It looks like this caused some bugs? https://github.com/go-task/task/issues/1317#issuecomment-2620027809

Reply:
Probably not, it was just always wrong and I can't see why it would cause what was described in the comment.